.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+### Changes proposed in this Pull Request:
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+Closes # .
+
+_Replace this with a good description of your changes & reasoning._
+
+
+### Screenshots:
+
+<!--- Optional --->
+
+
+### Detailed test instructions:
+<!-- Add detailed instructions for how to test that this PR fixes the issue and confirm that it doesn't break any other features :) -->
+
+1. 
+2. 
+3. 
+
+
+### Additional details:
+
+<!--
+Optional.
+Enter a summary of all changes in this Pull Request, which will be added to the changelog if accepted.
+Each line should start with change type prefix`(Fix|Add|…) - `, for example:
+> Break - A change breaking previous API or functionality.
+> Add - A new feature, function or functionality was added.
+> Update - Big changes to something that wasn't broken.
+> Fix - Took care of something that wasn't working.
+> Tweak - Small change, that isn't actually very important.
+> Dev - Developer-facing only change.
+> Doc - Updated customer or developer facing documentation
+
+If you remove the "Changelog entry" header, the Pull Request title will be used as the changelog entry.
+
+Add the `changelog: none` label if no changelog entry is needed.
+-->
+### Changelog entry
```

Comment:
Will this repo be released via the `woorelease` tool? If not, maybe these descriptions can be omitted.

Reply:
Good point, thanks :) Initially, I thought it would be nice to keep the template on our habits consistent across repos. But as we're not gonna use woorelease, this section will not be used in the changelog at all and could only lead to confusion.